content/blog/joining-ggplot2/index.Rmd

```diff
@@ -0,0 +1,118 @@
+---
+output: hugodown::hugo_document
+
+slug: joining-ggplot2
+title: Joining the ggplot2 team
+date: 2025-01-09
+author: Teun van den Brand
+description: >
+    I joined the ggplot2 team and would like to share the experience.
+
+photo:
+  url: https://unsplash.com/photos/person-holding-green-plant-on-black-pot-CbZh3kaPxrE
+  author: Jonathan Kemper
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [other] 
+tags: []
```

Comment:
Not sure what to add as tag. I guess 'internship' it the closest pre-existing one?

Reply:
yeah, I think that's ok even though you strictly aren't an intern. In spirit we are all alike :-)